tools/autograd/derivatives.yaml

```diff
@@ -333,6 +333,10 @@
   p: zeros_like(p)
   result: self_t.zero_()
 
+- name: bernoulli.p(Tensor self, float p, *, Generator? generator=None) -> Tensor
+  self: zeros_like(grad)
+  result: auto_element_wise
```

Comment:
The behavior today is that bernoulli.p returns a Tensor without requires_grad. Does this PR change the behavior by giving it a backwards formula?
```
In [6]: x = torch.randn(3, 3, requires_grad=True)

In [7]: y = torch.bernoulli(x, 0.5)

In [8]: y
Out[8]:
tensor([[1., 1., 0.],
        [0., 1., 0.],
        [1., 0., 0.]])

```

Reply:
Correct, please mark `output_differentiability = [False]` here to preserve that behavior

Reply:
whoops, thanks :)